R/tt_dotabulation.R

```diff
@@ -1250,8 +1250,11 @@ setMethod("set_def_child_ord", "VarLevWBaselineSplit",
             vals <- levels(relevel(droplevels(vals), bline))
         }
     }
-
-    stopifnot(bline %in% vals)
+    if (!bline %in% vals) {
+        stop(paste0(
+            '"', bline, '"', ' was not present in the levels of ', spl_payload(lyt), ' in the data.'
```

Comment:
We still want to say that the missing level was supposed to be the reference group so that the user knows where to look to fix the error. Please do something along the lines of 

"Reference group (<bline>) was not present in levels of <spl_payload(lyt)> in the data."

Reply:
Error message updated accordingly.